package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sia",
-  "version": "1.0.0",
+  "version": "1.16.0",
```

Comment:
Change isn't really necessary since we're not publishing this package

Reply:
The version was not reflecting the release number which is confusint.